README.md

```diff
@@ -62,6 +62,10 @@ curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/ajeetdsou
 If you want the interactive fuzzy selection feature, you will also need to install
 [`fzf`](https://github.com/junegunn/fzf.git).
 
+#### Packaging status
+
+Fedora/CentOS [COPR](https://copr.fedorainfracloud.org/coprs/atim/zoxide/): `sudo dnf copr enable atim/zoxide -y && sudo dnf install zoxide`
```

Comment:
I would personally rather see this split into separate lines, as well as drop the "automatic"-ness of the `-y` flag:

```shell
dnf copr enable atim/zoxide
dnf install zoxide
```

Additionally, if the user needs to run this as root, they should do it themselves and not need to be told by us.